video/vaapi.c

```diff
@@ -309,10 +308,12 @@ static const struct va_native_display *const native_displays[] = {
 static struct AVBufferRef *va_create_standalone(struct mpv_global *global,
         struct mp_log *log, struct hwcontext_create_dev_params *params)
 {
+    const char *path;
+    mp_read_option_raw(global, "vaapi-device", &m_option_type_string, &path);
```

Comment:
As pointed out in the original PR, this talloc__strdup(NULL,...)'s the valu for string type. For this (and other reasons) mp_read_option_raw is frowned upon for strings and `mp_get_config_group` is instead prefered (see `filters/f_swresample.c`)